frontend/src/store/sagas/addresses.js

```diff
@@ -0,0 +1,37 @@
+import {
+  put,
+  // call
+} from 'redux-saga/effects'
+import { fetchAddresses } from '../api/addresses'
+import {
+  GET_ALL_ADDRESSES_SUCCESS,
+  GET_ALL_ADDRESSES_FAILURE,
+  GET_SEARCH_ADDRESSES_SUCCESS,
+  GET_SEARCH_ADDRESSES_FAILURE,
+} from '../actions/addresses'
+import defaultErrorHandler from './defaultErrorHandler'
+
+export function* fetchAddressesSaga(action) {
```

Comment:
I think we  should start adopting this naming convention. This would definitely help searching files in the editor. Also, you can know which type of function you are dealing with just by naming convention. 

Nice one! :+1: :+1: